authority/voting/client/client.go

```diff
@@ -374,6 +372,9 @@ func (c *Client) Get(ctx context.Context, epoch uint64) (*pki.Document, []byte,
 		c.log.Errorf("voting/Client: Get() invalid consensus document: %s", err)
 		return nil, nil, err
 	}
+	if doc.SphinxGeometry == nil {
+		panic("received pki doc with nil SphinxGeometry")
+	}
```

Comment:
Do we need to validate that it's a *sane* geometry (positive number of hops etc?)
Doesn't this check belong in `pki.VerifyAndParseDocument` (then it could also fail gracefully instead of panicing)?

Reply:
Yes we should have a Validate method on Geometry to make sure it is sane. I'll add that.

Reply:
Okay! I've added the Validate method... and call it at the appropriate places